lib/internal/fs.js

```diff
@@ -52,7 +53,7 @@ function stringToFlags(flag) {
     case 'xa+': return O_APPEND | O_CREAT | O_RDWR | O_EXCL;
   }
 
-  throw new Error('Unknown file open flag: ' + flag);
+  throw new errors.Error('ERR_UNK_FILE_OPEN_FLAG', flag);
```

Comment:
I recommend to use the `ERR_INVALID_OPT_VALUE` error type for both cases here instead of creating new ones.